### DISCUSSION TITLE
Possible to use Liquid syntax on Manage Content page in order to differentiate between different items

### DISCUSSION THREAD

**Discussion by jameswoodley** [OrchardCMS/OrchardCore#15806] [Q&A]
We have a number of items that have the same Title, but we have a Search Field within the content type which we can use in the content selectors, but it seems not in the Manage Content screen.. see screenshot for example, they have title all as "Action Steps", but within they have a Search Field which is different for each item. Can I use that value for this page?

![image](https://github.com/OrchardCMS/OrchardCore/assets/52610563/72e1c090-2e70-4582-86ef-21e896f00b7a)
![image](https://github.com/OrchardCMS/OrchardCore/assets/52610563/5de18483-24e8-44d5-a2a4-0c9bf795e89f)

**Comment by Piedone** [comment]
Full-text search can provide this: https://docs.orchardcore.net/en/latest/docs/reference/modules/Contents/#full-text-search-for-admin-ui

However, why do you have the same title for every content item? Note that you can show them with seemingly the same title on the frontend even when saving different Title Part values for them. Additionally, you can set the `DisplayText` (what the list actually shows) differently from the Title.

**Comment by jameswoodley** [comment]
The title is the same because they are the same item in different sections of our site, so Category A has an Action Steps, Category B has an Action Steps etc etc

I'll check out those docs, but when I tried with full text search, the text didn't change, i'll try again making sure I've not messed something up

**Reply by Piedone** [reply]
Next time please use this textbox to write replies to messages in the discussion board: 
![image](https://github.com/OrchardCMS/OrchardCore/assets/1976647/05c678ea-95fb-4731-b940-e846af5012cc)
